### DISCUSSION TITLE
When will come a gdevelop5 non beta version?

### DISCUSSION THREAD

**Discussion by OverloadedTech** [4ian/GDevelop#2564] [Archive]
Hello, where will come the gdevelop5 non-beta software??

**Comment by Bouh** [comment]
[A discussion about it exist on Trello](https://trello.com/c/vMX7TOkv/130-remove-the-beta-from-the-gd5-version-number)

**Reply by 4ian** [reply]
Locking this as the discussion is already on Trello :)
